src/Maybe/Extra.elm

```diff
@@ -1,5 +1,5 @@
 module Maybe.Extra exposing
-    ( isJust, isNothing, join, filter
+    ( isJust, isNothing, join, filter, guarded
```

Comment:
If we do merge this function, when should it get published? I'm not a fan of frequent small releases, especially for very popular packages like this one. Would it be okay to sit on this until there's other changes that need to go out? Or would it be better to do a 5.2.0 release just for this one function?

Sorry, the right way to handle this would have been to resolve #39 before releasing 5.1.0, but I put it off.

(Attaching to a line of code so this discussion can be threaded separately from the rest of the discussion.)

Reply:
I have no strong opinion either way. #39 has been open for nearly 2 years, a little longer won't change anything.